main.py

```diff
@@ -28,11 +29,20 @@ async def create_project(request: web.Request):
     # fetch eth
     tier1_expected_amount = get_eth_amount(min_payment_amount_tier1)
     tier2_expected_amount = get_eth_amount(min_payment_amount_tier2)
+    tier1_expected_amount_ablock = get_ablock_amount(min_payment_amount_tier1 * discount_ablock)
+    tier2_expected_amount_ablock = get_ablock_amount(min_payment_amount_tier2 * discount_ablock)
+    tier1_expected_amount_aablock = get_aablock_amount(min_payment_amount_tier1 * discount_aablock)
+    tier2_expected_amount_aablock = get_aablock_amount(min_payment_amount_tier2 * discount_aablock)
+
     if not tier1_expected_amount or not tier2_expected_amount:
         return web.json_response({
             'error': 'Internal Server Error: Failed to get coinbase ETH prices, please try again'
```

Comment:
Oh, I guess I was wrong about the possibility "coinbase" could be written to the log. Still, here is a possible error response (line 33/39) of `main.py` where "coinbase" could be given in an error response.